docs/EN_US/HPCCDataHandling/DH-Mods/DH-Mod1.xml

```diff
@@ -34,6 +34,74 @@
     that server to enable file sprays and desprays.</para>
   </sect1>
 
+  <sect1 id="FileMovement">
+    <title>File Movement</title>
+
+    <para>The ESP file service can request whether an import (spray), despray,
+    or copy command is a <emphasis role="bold">push</emphasis> or a <emphasis
+    role="bold">pull</emphasis> command.</para>
+
+    <para>These processes run as DFU workunits and communicate with service
+    components, either running as dynamically launched ftslave processes (the
+    default behavior on Bare-Metal deployments), or as tasks running on a
+    dafilesrv service (the only option on a Containerized deployment).</para>
+
+    <itemizedlist>
+      <listitem>
+        <para><emphasis role="bold">pull</emphasis> means that the commands
+        will notionally run per target part and pull from the source.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="bold">push</emphasis> means the command will
+        notionally run per source part and push to the target.</para>
+      </listitem>
+    </itemizedlist>
+
+    <para>In Bare-Metal deployments, where the compute is coupled to the data,
+    ftslave processes are launched on either the source or the target nodes
+    depending on whether pushing or pulling.</para>
+
+    <para>In a Containerized system, the commands always run on a dedicated
+    dafilesrv spray-service where neither the source nor the target are
+    local.</para>
+
+    <para>The default setting in a bare-metal system is useFtSlave=true, an
+    option of dfuserver, but can be configured off so Bare-Metal will use
+    dafilesrv as a service, too.</para>
+
+    <para>It is generally recommended to leave the decision to whether to pull
+    or push up to the platform. In Bare-Metal, it may decide that it has to
+    push because running ftslave on the target side is not supported such as
+    when the the Landing Zone is a Windows server. Another case is when the
+    system might choose to pull when it is more efficient to pull vs. push. In
```

Comment:
> Another case is when the system might choose to pull when it is more efficient to pull vs. push.

Is a bit of a mouth full, perhaps change to:

It may also choose to pull when it concludes it is more efficient to do so.